### DISCUSSION TITLE
How to add torch.compile() in LightningModule

### DISCUSSION THREAD

**Discussion by mactavish91** [Lightning-AI/pytorch-lightning#15896] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
I want to use pytorch2.0 to accelerate training, is there any method?

**Comment by akihironitta** [comment]
You can simply pass your LM to `torch.compile(...)`:
```python
compiled_model = torch.compile(your_lightning_module)
trainer.fit(compiled_model)
```

Issues/PRs/discussions labelled "pytorch 2.0" might have useful information for you :)
